ilastik/utility/bioimageio_utils.py

```diff
@@ -343,5 +343,11 @@ def check_single_image_min_shape(self, tensor_id: str, target_vigra_shape: Tagge
             ), f"Axis {axis_id} not an expected bioimage io axis string {SPEC_TO_VIGRA.keys()}."
 
             target_axis_size = spec_shape[axis_id]
-            if target_axis_size < min_size:
-                raise ValueError(f"Incompatible axis {axis}: {target_axis_size} < {min_size}")
+            if isinstance(axis, ChannelAxis):
+                if target_axis_size != min_size:
+                    raise ValueError(
+                        f"Incompatible axis {axis}: Number of channels in data: {target_axis_size} != Number of channels expected by model: {min_size}"
```

Comment:
```suggestion
                    raise ValueError(
                        f"Selected input image has {target_axis_size} channels, but model expects {min_size}"
```

![image](https://github.com/user-attachments/assets/49679b33-a2f1-4fa5-bc18-c65a0d17d871)

Reply:
Sounds reasonable coming with a two channel image?

Reply:
but yeah the other one.. .lets see.

Reply:
in any case, tests were not passing for similar reasons, some changes in the tiktorch api. Fixed now